app/enterprise/0.33-x/admin-gui/configuration/property-reference.md

```diff
@@ -1,24 +1,56 @@
 ---
-title: Setting custom URLs for the Admin GUI and Admin API
+title: Property Reference
 book: admin_gui
-chapter: 5
+chapter: 6
 ---
 
-# Property Reference
+# Admin GUI Property Reference
+
+This document describes configuration options for the Kong Admin GUI.
+
+
+## admin_uri
+
+**Default:** `NONE` (auto generated)
+
+**Description:**  
+The URL on which the Admin API is accessible. By default the Admin GUI 
+assumes the Admin API is on the same host, and will use the window request 
+host and resolved listener port depending on the requested protocol. This 
+is where the Admin GUI finds the Admin API when `admin_gui_auth` (see below) 
+is not enabled.
+
+Note that this property accepts a value with or without protocol. If you 
+specify the protocol, be sure that it is the same as the one you set in 
+`admin_gui_url`, to prevent rendering errors due to mixed content.
+
+Note: This property will be renamed `admin_url` in a future release, to be 
+consistent with other URL properties in `kong.conf`. Protocol will be 
+required at that time. It is strongly recommended that your Admin API be 
+on a secure port.
+
+**Example:**
+
+```
+admin_uri = https://127.0.0.1:8001
```

Comment:
`admin_api_uri` and if we are demonstrating https in the example maybe we should use the default SSL port 8444